Directory.Build.props

```diff
@@ -13,6 +13,12 @@
 
     <!-- Allow references to unsigned assemblies (like MAUI) from signed projects -->
     <NoWarn>$(NoWarn);CS8002</NoWarn>
+
+    <!-- We need to support old stuff. Applications should definitely address these advisory warnings though. -->
+    <NoWarn>$(NoWarn);NU1903</NoWarn>
```

Comment:
Since it's a major, it's a good time to review the lowest we support and bump some a bit (not necessarily to latest)